sources/logdog/Cargo.toml

```
@@ -15,7 +15,7 @@ datastore = { path = "../api/datastore", version = "0.1.0" }
 flate2 = "1.0"
 glob = "0.3"
 models = { path = "../models", version = "0.1.0" }
-reqwest = { version = "0.11.1", default-features = false, features = ["blocking", "rustls-tls"] }
+reqwest = { version = "0.11.1", default-features = false, features = ["blocking", "rustls-tls", "rustls-tls-native-roots"] }
```

The [reqwest docs](https://docs.rs/reqwest/latest/reqwest/#optional-features) imply we should just use:
```suggestion
reqwest = { version = "0.11.1", default-features = false, features = ["blocking", "rustls-tls-native-roots"] }
```

Since `rustls-tls` is equivalent to `rustls-tls-webpki-roots`.